test/common/markdown/markdown-link.tests.ps1

```diff
@@ -1,15 +1,14 @@
-﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What is changed here? BOM?

Reply:
There was some kind of red dot, I got rid of it.

Reply:
I see that the file had BOM before the fix.